src/facade/conn_context.h

```diff
@@ -47,17 +47,11 @@ class ConnectionContext {
     return res;
   }
 
-  void SendError(std::string_view str, std::string_view type = std::string_view{}) {
-    rbuilder_->SendError(str, type);
-  }
+  virtual void SendError(std::string_view str, std::string_view type = std::string_view{});
```

Comment:
The only problem I see here is that these can be skipped if the caller uses `reply_builder()->SendError` which we do spuriously around the codebase. From my understanding we want this to cover lua scripts so it's worth double checking that we actually use this interface rather than the indirect way `reply_builder()->SendError()` which bypasses this flow

Reply:
good catch, fixed.